.env.example

```diff
@@ -1,27 +1,38 @@
-PUBLIC_ALGOLIA_APP_ID=""
-PUBLIC_ALGOLIA_API_KEY=""
-PUBLIC_ALGOLIA_INDEX=""
-PUBLIC_GOOGLE_ANALYTICS_MEASUREMENT_ID=""
-PUBLIC_YOUTUBE_API_KEY=""
-PUBLIC_YOUTUBE_CHANNEL_ID=""
-PUBLIC_CONTENTFUL_OAUTH_ENDPOINT="https://be.contentful.com/oauth/authorize"
-PUBLIC_CONTENTFUL_OAUTH_CLIENT_ID=""
-PUBLIC_CONTENTFUL_OAUTH_CLIENT_REDIRECT_URI=""
 ALGOLIA_API_KEY=""
 AUTH_TOKEN=""
-CHROMATIC_PROJECT_TOKEN=""
 CONTENTFUL_DEFAULT_ENVIRONMENT=""
 CONTENTFUL_DELIVERY_API_TOKEN=""
+CONTENTFUL_IMAGE_API_ENDPOINT=""
+CONTENTFUL_MANAGEMENT_API_ENDPOINT=""
 CONTENTFUL_PREVIEW_API_TOKEN=""
 CONTENTFUL_SPACE_ID=""
-CONTENTFUL_IMAGE_API_ENDPOINT="https://images.ctfassets.net"
-CONTENTFUL_MANAGEMENT_API_ENDPOINT="https://api.contentful.com"
+KV_REST_API_READ_ONLY_TOKEN=""
+KV_REST_API_TOKEN=""
+KV_REST_API_URL=""
 KV_URL="redis://localhost"
+PUBLIC_ALGOLIA_API_KEY=""
+PUBLIC_ALGOLIA_APP_ID=""
+PUBLIC_ALGOLIA_INDEX=""
+PUBLIC_CONTENTFUL_OAUTH_CLIENT_ID=""
+PUBLIC_CONTENTFUL_OAUTH_CLIENT_REDIRECT_URI=""
+PUBLIC_CONTENTFUL_OAUTH_ENDPOINT=""
+PUBLIC_GOOGLE_ANALYTICS_MEASUREMENT_ID=""
+PUBLIC_VERCEL_SPEED_INSIGHTS_ANALYTICS_ID=""
```

Comment:
Pretty sure this one is defunct but I'm going to wait to handle this later (since we need to switch over to using the `@vercel/speed-insights` package).

Other things removed were:
* `CHROMATIC_PROJECT_TOKEN` since we stopped using this halfway through development
* `LDAF_SLACK_WEBHOOK_URL` (or something like that) since we deprecated the alerts channel in favor of the Vercel logs